os/se/ameba/rtl_security_api_wrapper_tz.c

```diff
@@ -79,6 +81,11 @@ static void ns_setstatusbits(u32 NewState)
 	device_mutex_unlock(RT_DEV_LOCK_FLASH);
 }
 
+static void ns_printf(const char *input)
+{
+	sedbg("%s \n", input);
```

Comment:
Could you remove a space to reduce unnecessary data?
```
sedbg("%s\n", input);
```

Reply:
Done modify